CMakeLists.txt

```diff
@@ -98,7 +117,7 @@ function(trusted_firmware_build)
     SOURCE_DIR ${ZEPHYR_TFM_MODULE_DIR}/trusted-firmware-m
     BINARY_DIR ${TFM_BINARY_DIR}
     CMAKE_ARGS -DTFM_TOOLCHAIN_FILE=${ZEPHYR_TFM_MODULE_DIR}/${TFM_TOOLCHAIN_FILE}
-               -DTFM_PLATFORM=${TFM_BOARD}
+               -DTFM_PLATFORM=${TFM_PLATFORM_DIR}
```

Comment:
Can't this just be `-DTFM_PLATFORM=${CONFIG_TFM_BOARD}

Reply:
@galak, cf relative discussion here: https://github.com/zephyrproject-rtos/zephyr/pull/30062#discussion_r524413356